YololCompetition/Program.cs

```diff
@@ -39,11 +39,15 @@ public static async Task Main(string[] args)
             var commands = provider.GetService<CommandService>();
             await commands.AddModulesAsync(Assembly.GetExecutingAssembly(), provider);
 
+            var messages = provider.GetService<IMessages>();
+
             var bot = provider.GetService<DiscordBot>();
             await bot.Start();
 
             Console.WriteLine("Bot Started");
             await provider.GetRequiredService<IScheduler>().Start();
+
+            messages.StartMessageWatch();
```

Comment:
Moved the startmessagewatch to below where the bot has announced that it is started.